src/tests/object-mother/transaction-result.ts

```diff
@@ -3047,4 +3047,298 @@ export const transactionResultMother = {
       'tx-type': TransactionType.axfer,
     })
   },
+  ['testnet-QY4K4IC2Z5RQ5OM2LHZH7UAFJJ44VUDSVOIAI67LMVTU4BHODP5A']: () => {
```

Comment:
Should we break this file up?

Reply:
We could. Potentially we could store each transaction as a ts file. This would then simply expose those files to the tests.